site/content/en/docs/tasks/run_plain_pods.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Run A Plain Pod"
+title: "Run Plain Pods"
 date: 2023-09-27
 weight: 6
 description: >
-  Run a Kueue scheduled Pod.
+  Run Jobs represented by plain pods, either single pods, or pod groups.
```

Comment:
maybe
```suggestion
  Run a group of Pods as a Kueue managed Job.
```

Reply:
I would like to emphasize that you can run either a single pod, or a group of pods.
Maybe: "Run a single Pod, or a group of Pods, as a Kueue-managed Job.". WDYT?